ONSdigital/dp-renderer
# Feature/Breadcrumbs on census DLP

### What

- Added breadcrumbs from the [design system](https://ons-design-system.netlify.app/components/breadcrumbs/)
- Migrated iconography from `dp-frontend-dataset-controller`

### How to review

- Check the design against the [design system](https://ons-design-system.netlify.app/components/breadcrumbs/) and Figma
- Check that pages not using the design system (e.g CMD) still have the old style breadcrumbs 

#### Old/current on CMD
<img width="703" alt="Screenshot 2021-10-07 at 13 41 56" src="https://user-images.githubusercontent.com/19624419/136386049-455a3fb5-c0b9-4d0b-91a7-daefad76f2fd.png">

#### New
<img width="703" alt="Screenshot 2021-10-07 at 13 42 54" src="https://user-images.githubusercontent.com/19624419/136386173-4f941913-c8b0-499b-945e-f7311666636c.png">

**Note** due to a conflict with Sixteens, hovering over the links is not as clear as it should be and will be fixed with the modular system

### Who can review

Frontend dev
